Beam/resources/assets/js/remplib.js

```diff
@@ -119,6 +132,19 @@ remplib = typeof(remplib) === 'undefined' ? {} : remplib;
             window.addEventListener("campaign_showtime", this.syncOverridableFields);
             window.addEventListener("campaign_showtime", this.syncFlags);
             window.addEventListener("beam_event", this.incrementSegmentRulesCache);
+
+            if (this.config.tracker.readingProgress.enabled) {
```

Comment:
This condition causees `Cannot read property 'tracker' of undefined`. See how other parts are reading the configured parameters.